features/sessions/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ plugins {
     id(BuildPlugins.kotlinAndroid)
     id(BuildPlugins.kotlinAndroidExtensions)
     id(BuildPlugins.ktlintPlugin)
+    id("kotlin-android")
```

Comment:
Remove this Reference SInce it Already has been added at the top.

Also when adding plugins, and/ or dependencies, please use the variables from `buildSrc` `Dependencies. kt` file